.gitignore/test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Kindly ensure you test.md file is ignored by .gitignore, kindly change the `.gitignore` folder to a file so that it can ignore files listed in it.